src/gui/mzroll/mainwindow.cpp

```diff
@@ -2953,40 +2962,44 @@ void MainWindow::createToolBars() {
 	ionizationModeLabel->setFrameShape(QFrame::Panel);
 	ionizationModeLabel->setFrameShadow(QFrame::Raised);
 
-	
-	ionChargeBox = new QSpinBox(hBox);
-	ionChargeBox->setValue(settings->value("ionChargeBox").toInt());
+    ionChargeBox = new QSpinBox(hBox);
+    connect(ionChargeBox, SIGNAL(valueChanged(int)), this, SLOT(setTotalCharge()));
+    ionChargeBox->setValue(settings->value("ionChargeBox").toInt());
 
-	quantType = new QComboBox(hBox);
-	quantType->addItem("AreaTop");
-	quantType->addItem("Area");
+    quantType = new QComboBox(hBox);
+    quantType->addItem("AreaTop");
+    quantType->addItem("Area");
     quantType->addItem("Height");
-    quantType->addItem("AreaNotCorrected"); //TODO: Sahil-Kiran, Added while merging mainwindow
+    quantType->addItem("AreaNotCorrected");
     quantType->addItem("AreaTopNotCorrected");
     quantType->addItem("Retention Time");
-	quantType->addItem("Quality");
-	quantType->setToolTip("Peak Quntitation Type");
```

Comment:
Since this part of the code is being touched, could you please correct the typo? :P

Reply:
Missed this. Correcting.